src/main/thrift/tag.thrift

```diff
@@ -74,6 +74,12 @@ struct PodcastMetadata {
 
     /** The Pocket Casts url for the podcast **/
     14: optional string pocketCastsUrl
+
+    /** Whether or not episodic art should be enabled */
+    15: optional bool episodicArtworkEnabled;
+
+    /** The date from which episodic art should be enabled */
```

Comment:
What sort of date number should this be? Assuming Unix epoch in ms, but it'd be good to be specific.

Reply:
That's right. I'll add a comment!